### PR TITLE
Update resource id support.

### PR DESCRIPTION
**Pull request by Temppus**

### CODE REVIEW

File: sfcli/SmartFace.Cli/Core/Domain/ResourceIds.cs [hunk 1]
```diff
@@ -2,19 +2,23 @@
 {
     public static class ResourceIds
     {
-        public static class TemplateGenerator
+        public class TemplateGeneratorResourceId
```
Comment (blaskoa) [inline sfcli/SmartFace.Cli/Core/Domain/ResourceIds.cs:5]: is this file still needed?

File: sfcli/SmartFace.Cli/Infrastructure/ApiImplementation/CamerasRepository.cs [hunk 2]
```diff
@@ -77,46 +75,12 @@ private Camera UpdateCameraData(Camera originalCamera, CameraRequestData updateD
             updatedCamera.FaceDetectorConfig.MaxFaceSize = updateData.TrackMaxFaceSize ?? updatedCamera.FaceDetectorConfig.MaxFaceSize;
             updatedCamera.FaceDetectorConfig.MinFaceSize = updateData.TrackMinFaceSize ?? updatedCamera.FaceDetectorConfig.MinFaceSize;
 
-            //TODO when the API takes the resources as string, maybe just forward them, no mapping needed
-            updatedCamera.TemplateGeneratorResourceId = MapTemplateResourceId(updateData.TemplateGeneratorResourceId) ?? updatedCamera.TemplateGeneratorResourceId;
-            updatedCamera.FaceDetectorResourceId = MapFaceDetectorResourceId(updateData.FaceDetectorResourceId) ?? updatedCamera.FaceDetectorResourceId;
+            updatedCamera.TemplateGeneratorResourceId = updateData.TemplateGeneratorResourceId;
```
Comment (blaskoa) [inline sfcli/SmartFace.Cli/Infrastructure/ApiImplementation/CamerasRepository.cs:78]: I think we should follow the same pattern with `updatedCamera.TemplateGeneratorResourceId = updateData.TemplateGeneratorResourceId ?? updatedCamera.TemplateGeneratorResourceId;`, so that we don't overwrite the resource ID when updating other fields

File: sfcli/SmartFace.Cli/Core/Domain/ResourceIds.cs (hunk 1 above)
Comment (Temppus) [reply sfcli/SmartFace.Cli/Core/Domain/ResourceIds.cs:5]: removed

File: sfcli/SmartFace.Cli/Infrastructure/ApiImplementation/CamerasRepository.cs (hunk 2 above)
Comment (Temppus) [reply sfcli/SmartFace.Cli/Infrastructure/ApiImplementation/CamerasRepository.cs:78]: done
